watch-library/simulator/watch/watch_deepsleep.c

```diff
@@ -35,6 +35,7 @@ static uint32_t watch_backup_data[8];
 
 void watch_register_extwake_callback(uint8_t pin, ext_irq_cb_t callback, bool level) {
     if (pin == BTN_ALARM) {
+        watch_enable_external_interrupts();
```

Comment:
If a watch app explicitly uses `watch_register_extwake_callback()` to wake with the Alarm button, we need to enable interrupts (a simulator implementation detail).

Reply:
right because on the hardware, that interrupt is part of the RTC's responsibilities, makes sense!